### DISCUSSION TITLE
More information for import errors

### DISCUSSION THREAD

**Discussion by alex-pahdo** [godotengine/godot-proposals#5632] [Editor]
This came up while debugging a Github workflow that failed to import almost all of our asset files. Eventually, after following the code and thinking through it, I realized the problem was that the workflow hadn't checked out LFS files, so the files were pointers instead of the actual assets. For a wav file, we get ERR_FILE_UNRECOGNIZED because the file didn't start with a RIFF header. If it printed the first 4 characters ('vers') or even checked for the word 'version' to suggest that it may be a pointer file, I would have had a lead much earlier on. 

So maybe when the file is unrecognized, we could print a report of the size, encoding, and if it's a printable format like UTF-8, the error message could include the mismatched header, or the first line if appropriate.

**Comment by Calinou** [comment]
Feel free to open a pull request for this :slightly_smiling_face: 

WAV importer code is here: https://github.com/godotengine/godot/blob/master/editor/import/resource_importer_wav.cpp
